internal/otlp/marshaler.go

```diff
@@ -18,17 +18,32 @@ import (
 	"go.opentelemetry.io/collector/internal/model"
 )
 
-// NewJSONTracesMarshaler returns a model.TracesMarshaler to decode from OTLP json bytes.
+// NewJSONTracesMarshaler returns a model.TracesMarshaler. Marshals to OTLP json bytes.
 func NewJSONTracesMarshaler() model.TracesMarshaler {
 	return model.NewTracesMarshaler(newJSONEncoder(), newFromTranslator())
 }
 
-// NewJSONMetricsMarshaler returns a model.MetricsMarshaler to decode from OTLP json bytes.
+// NewJSONMetricsMarshaler returns a model.MetricsMarshaler. Marshals to OTLP json bytes.
 func NewJSONMetricsMarshaler() model.MetricsMarshaler {
 	return model.NewMetricsMarshaler(newJSONEncoder(), newFromTranslator())
 }
 
-// NewJSONLogsMarshaler returns a model.LogsMarshaler to decode from OTLP json bytes.
+// NewJSONLogsMarshaler returns a model.LogsMarshaler. Marshals to OTLP json bytes.
 func NewJSONLogsMarshaler() model.LogsMarshaler {
 	return model.NewLogsMarshaler(newJSONEncoder(), newFromTranslator())
 }
+
+// NewProtobufTracesMarshaler returns a model.TracesMarshaler. Marshals to OTLP binary protobuf bytes.
+func NewProtobufTracesMarshaler() model.TracesMarshaler {
```

Comment:
@jrcamp should this be `Protobuf` or `Pb` or `Proto`?

Reply:
I'd say `Protobuf`

Reply:
Ok